neqo-http3/benches/streams.rs

```diff
@@ -76,21 +76,18 @@ fn connect() -> (Http3Client, Http3Server) {
 fn criterion_benchmark(c: &mut Criterion) {
     fixture_init();
 
-    for (streams, data_size) in [
-        (1, 1),
-        (1000, 1),
-        (10000, 1),
-        (1, 1000),
-        (100, 1000),
-        (1000, 1000),
-    ] {
+    for (streams, data_size) in [(1_000, 1), (1_000, 1_000)] {
         let mut group = c.benchmark_group(format!("{streams} streams of {data_size} bytes"));
+
+        // High variance benchmark. Increase default sample size (100).
+        group.sample_size(500);
```

Comment:
The core goal of this pull request, reducing the number of false positive performance regressions.